libs/core/langchain_core/messages/tool.py

```diff
@@ -39,6 +54,14 @@ class ToolMessage(BaseMessage):
     type: Literal["tool"] = "tool"
     """The type of the message (used for serialization). Defaults to "tool"."""
 
+    raw_output: Any = None
```

Comment:
can this be `Any` and support serialization at the same time?

Reply:
message will fail to deserialize if this isn't serializable (see unit test)